[Bug]: Latest node 18 release is breaking the dashboard

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Based on the comments on https://github.com/opendatahub-io/odh-manifests/pull/770

We are experiencing some issues with the latest version of the dashboard

```bash
> ods-dashboard-backend@1.0.0 start
> NODE_ENV=production node ./dist/server.js --log=1 --registry=localhost:50051

npm ERR! code EACCES
npm ERR! syscall open
npm ERR! path /opt/app-root/src/.npm/_cacache/tmp/a99da38d
npm ERR! errno -13
npm ERR! 
npm ERR! Your cache folder contains root-owned files, due to a bug in
npm ERR! previous versions of npm which has since been addressed.
npm ERR! 
npm ERR! To permanently fix this problem, please run:
npm ERR!   sudo chown -R 1000690000:0 "/opt/app-root/src/.npm"

npm ERR! Log files were not written due to an error writing to the directory: /opt/app-root/src/.npm/_logs
npm ERR! You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
```

We should revert to the latest image to avoid this issue until we find a proper solution

### Expected Behavior

Dashboard is deployed successfully.

### Steps To Reproduce

_No response_

### Workaround (if any)

_No response_

### What browsers are you seeing the problem on?

_No response_

### Open Data Hub Version

_No response_

### Anything else

_No response_

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,7 +2,7 @@
 ARG SOURCE_CODE=.
 
 # Use ubi8/nodejs-18 as default base image
-ARG BASE_IMAGE="registry.access.redhat.com/ubi8/nodejs-18:latest"
+ARG BASE_IMAGE="registry.access.redhat.com/ubi8/nodejs-18:1-32.1679484519"
 
 FROM ${BASE_IMAGE} as builder
 

